app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -160,6 +213,27 @@ function AcuantCaptureCanvas({
     };
   }, [isReady]);
 
+  useEffect(() => {
+    if (canvasRef.current) {
+      // Acuant SDK assigns a callback property to the canvas when it switches to its "Tap to
+      // Capture" mode (Acuant SDK v11.4.4, L158). Infer capture type by presence of the property.
+      defineObservableProperty(canvasRef.current, 'callback', () => {
+        setCaptureType(canvasRef.current?.callback ? 'TAP' : 'AUTO');
+      });
```

Comment:
what if we updated `defineObservableProperty` to pass `currentValue` to the callback?

```suggestion
      defineObservableProperty(canvasRef.current, 'callback', (callback) => {
        setCaptureType(callback ? 'TAP' : 'AUTO');
      });
```

Reply:
> what if we updated `defineObservableProperty` to pass `currentValue` to the callback?

I can't recall if I had some reason not to allow this, aside from just thinking it was easy enough to just pull it back off the object in scope, but I agree it could make it a bit easier to work with to simply pass it as an argument.

Reply:
> what if we updated `defineObservableProperty` to pass `currentValue` to the callback?

Updated in f6fe7e8 (+ 6c2e637)